web/Views/Landlord/Profile.cshtml

```diff
@@ -70,6 +70,13 @@
     else
     {
         <p>Charter status: Pending</p>
+        @if (Model.CurrentUserIsAdmin)
+        {
+            @using (Html.BeginForm("ApproveCharter", "Landlord", FormMethod.Post))
+            {
+                <input type="hidden" id="landlordId" name="landlordId" value=@Model.Id>
+                <button type="submit" class="btn custom-btn-primary">Approve Landlord</button>

```

Comment:
Is `custom-btn-primary` defined anywhere?

Reply:
Ah no, I was wondering why that was working weirdly - it's because I looked at some code from bootcamp to figure out how something worked and forgot to change that.